apps/alert_processor/lib/service_info/service_info_cache.ex

```diff
@@ -453,7 +453,13 @@ defmodule AlertProcessor.ServiceInfoCache do
            })}
 
         {:ok, schedule} ->
-          [departure_schedule | _t] = Enum.sort_by(schedule, & &1["attributes"]["departure_time"])
+          [departure_schedule | _t] =
+            schedule
+            |> Enum.sort_by(& &1["attributes"]["departure_time"], fn
+              _, nil -> true
+              nil, _ -> false
+              a, b -> a <= b
```

Comment:
suggestion: Since we use this exact same logic in two places, maybe it could be extracted to a descriptively-named helper? Then this could be something like:

```elixir
alias AlertProcessor.Helpers.Sort
# ...
[departure_schedule | _t] =
  Enum.sort_by(schedule, & &1["attributes"]["departure_time"], &Sort.nils_last/2)
```